aten/src/ATen/gen.py

```diff
@@ -156,6 +175,8 @@ def backend_to_devicetype(backend):
     ('QInt32', 'qint32', 'QInt32AccrealNotDefined', 'Qint32IsFloatingTypeNotDefined'),
 ]
 
+# whitelist used to filter op registrations for custom build
+op_registration_whitelist = set(options.op_registration_whitelist or [])
```

Comment:
nit: semantically, an empty white list would be more consistent if it meant to have **no** operators present, not **all** operators like you're using it here. I think it helps readability if `op_registration_whitelist` doesn't use `set([])` but `None` to disable the whitelist.

Reply:
Yes I agree - thought about it but didn't know how to write it as short as the current version :p
I'll update.

Reply:
changed to None